packages/components/src/components/date-picker/date-picker-native.jsx

```diff
@@ -38,6 +40,14 @@ const Native = ({
         if (typeof onBlur === 'function') {
             onBlur(e);
         }
+
+        setOldValue(e.target.value);
+    };
+
+    const updateRef = e => {
```

Comment:
can we rename `e` to `value`?. The name `e` gives the assumption that it's an event.

```
const updateRef = value => {
```

Reply:
Indeed, I thought it was an event passed 😄